src/components/renderer/renderer.tsx

```diff
@@ -26,10 +27,11 @@ export default class Editor extends React.Component<Props> {
 
     const runtime = vega.parse(props.vegaSpec);
 
-    Editor.view = new vega.View(runtime)
-      .logLevel(vega.Warn)
-      .initialize(Editor.chart);
-
+    Editor.view = new vega.View(runtime, {
```

Comment:
Use something like this instead

```
const runtime = vega.parse(props.vegaSpec);

    const loader = vega.loader();
    const originalHttp = loader.http;
    loader.http = async(url, options) => {
      try {
		return await originalHttp(url, {options, ...{baseURL: this.props.baseURL}});
      } catch {
        return await originalHttp(url, options);
      }
    };

    Editor.view = new vega.View(runtime, {
      loader: loader,
      logLevel: vega.Warn,
    })
    .initialize(Editor.chart);
```

I haven't tested this but I think this communicates the idea.

Reply:
I think there is some problem with `loader.http`, its baseURL option is not working. Instead `loader.load` is working fine.